source/includes/_errors.md

```diff
@@ -1,36 +1,68 @@
 ## Errors
+Retrieves information for errors and discrepancy in the Metislabs API. This should provide the user with information on why certain `Readings` or `Predictions` are missing.
+
+### Usage
+
+          | Value
+---------:|:-----
+__endpoint__ | `https://api.metislabs.tech/1.0/errors`
+__method(s)__ | `GET`
+
+#### Parameters
+
+The `Default` column specifies the value used when the user does not specify a parameter in the request.
+
+Field | Type | Default | Limitation | Description
+-----:|:----:|:---------:|:----------:|:-----------
+__organisation__ | string | NA | NA | The organisation that the user belongs to.
+__site__ | string | First site available to the user | NA | The site that the user wants to retrieve data from.
 
-* The Metislabs API returns the following error codes. All of the errors are returned in the standard format mentioned [here](#response-format). 
-* All of the responses will return a `payload` attribute of `nil`.
-* In the `message` attribute, there is a more verbose message detailing the situation and what went wrong.
 
 ```json
 {
-    "status": "page-not-found",
-    "message": "<404 Page Not Found>: Your request did not contain the correct URL or format, please try again.", 
-    "payload": None
+
+    "status_code": 200,
+    "message":"Errors",
+    "payload":[
+        {
+            "category":"ingestion",
+            "end_timestamp":2,
+            "message":"Unable to retrieve historian data.",
+            "start_timestamp":1
+        },
+        {
+            "category":"predict_handler",
+            "end_timestamp":2,
+            "message":"Response error from the /config endpoint",
+            "start_timestamp":1
+        }
+    ],
+    "status":"OK"
 }
 ```
 
-### Client based errors
+#### Response
 
-* Client based errors usually result from incorrect data input, wrong resources specified or unauthenticated access, if any of these occur, please check again how the resources are being accessed or used.
+ Attribute | Type | Value
+---------:|:----:|:-----
+__status__ | string | "OK"
+__message__ | string | "Errors"
+__status_code__ | int | 200
 
-Status | Status Code | Meaning
----------------: | :-----------: | :--------
-__url-parameter-error__ | 400 | The parameters supplied to the URL are incorrect or of a bad format.
-__authentication-required__ | 401 | Authentication is required to access resources at the endpoint.
-__permission-denied__ | 403 | The permissions level associated with the request was not sufficient to use the endpoint.
-__page-not-found__ | 404 | The URL used in the request does not exist on the API.
-__method-not-allowed__ | 405 | The HTTP method supplied in the request is not permitted on the endpoint.
+The `payload` attribute associated with a successful response will be available as an array of `JSONs`, each `JSON` will
```

Comment:
array of 'objects' might be better than 'JSONs'? since JSON expands to javascript object notation this reads a bit oddly

Reply:
(I think this might be in a few places)